portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,226 @@
-#content {
+body {
+  background-image: url('images/floral_bg.jpg');
+}
+
+.content {
+  margin-top: auto;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  padding: 20px;
+  max-width: 1000px;
+  background-color: white;
+  font-size: 20px;
+}
+
+h3 {
+  text-shadow: 1px 1px 3px #ccc;
+}
+
+.info-bar {
+  border-right: 1px solid #ccc;
+}
+
+#name {
+  font-weight: bold;
+  margin: 15px 0px;
 }
 
-#greeting-container {
+#location i {
+  font-size: 24px;
+}
+#location span {
+  font-size: 18px;
+  margin: 20px 0px;
+}
+
+#links {
   margin-top: 20px;
 }
+#links a {
+  display: block;
+  margin-top: 10px;
+  color: #7c7b8a;
+}
+#links a i {
+  font-size: 30px;
+}
```

Comment:
I think there needs to be a new line between each of these blocks.

Reply:
Same for here and below

Reply:
Alright